packages/flutter_tools/lib/src/asset.dart

```diff
@@ -248,7 +252,6 @@ class ManifestAssetBundle implements AssetBundle {
     // see _AssetDirectoryCache.
     final List<String> excludeDirs = <String>[
       assetDirPath,
-      getBuildDirectory(),
       if (flutterProject.ios.existsSync())
```

Comment:
Double checking here: The problem these solved is that we'd sometimes pickup variants from build output directories

Something like:

```
assets/foo.txt
build/bar/baz/intermediates/foo.txt
```

Would get picked up as a variant. But with this change, the `bar/baz/intermediates/` should no longer match the variant code right? 

If so, I believe you could remove all excludeDirs except for assetDirPath

Reply:
is `assetDirPath` used by the google3 version of the manifest? If not, you can g3fix. If it is used I would just leave it for now

Reply:
I was slightly inaccurate in my previous comment, but the idea is the same. It was used here exclusively as an exclude directory, and I removed `excludeDirs` altogether. With that gone, I removed it altogether.

To answer your question, yes, g3 doesn't have any special use for it.